spec/scrabble_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+require_relative '../lib/scrabble.rb'
+
+	describe Scrabble::Foo do
+		context "Must use a string of letters." do 
+			it "returns an empty string upon integer input" do 
+			foo = Scrabble::Foo.new
+			expect(foo.convert(0)).to eq("")
+			end
+		end
+
+		context "tallies letters to score the word" do
+			it "returns the value of 16 when user inputs quarter." do
+			expect(Scrabble::Foo.score("quarter")).to eq(16)
+			end
+		end
+
+		context "returns the word from a given array with highest score" do
```

Comment:
Between the pseudocode above and these tests, you're almost there. 

You might try this approach:

1. add the ```.to eq(...)``` bits to the ```expect()``` calls in these two tests
2. add a test for the seven-letter word special case
3. work on #highest_score_from until these tests pass